addons/Godot-MToon-Shader/inspector_mtoon.gd

```diff
@@ -80,10 +81,11 @@ const single_line_properties = {
 	"_RimTexture": "_RimColor",
 	"_EmissionMap": "_EmissionColor",
 	"_OutlineWidthTexture": "_OutlineWidth",
+	"_SphereAdd": "_MatcapColor",
 }
 
 const single_line_after_properties = {
-	"_SphereAdd": "_EmissionColor",
```

Comment:
Unused code?

Reply:
no i was trying to debug an issue with the color picker. this might have been important. has to do with the sort order of mtoon inspector properties